pkgs/top-level/all-packages.nix

```diff
@@ -14197,6 +14197,8 @@ with pkgs;
     inherit (perlPackages.override { pkgs = pkgs // { imagemagick = imagemagickBig;}; }) PerlMagick;
   };
 
+  imagej = callPackage ../applications/graphics/imagej { };
```

Comment:
Note that `imagej` here will refer to a set unlike most other toplevel bindings.  The evaluator also will normally not recurse into sets, unless instructed to with `recurseIntoAttrs`. What is the motivation for returning a set to begin with?